newm-server/src/main/kotlin/io/newm/server/staticcontent/StaticContentRoutes.kt

```diff
@@ -1,13 +1,8 @@
 package io.newm.server.staticcontent
 
-import io.ktor.server.http.content.resources
-import io.ktor.server.http.content.static
-import io.ktor.server.http.content.staticBasePackage
+import io.ktor.server.http.content.staticResources
 import io.ktor.server.routing.Routing
 
 fun Routing.createStaticContentRoutes() {
-    static("/contents") {
-        staticBasePackage = "static"
-        resources(".")
-    }
+    staticResources(remotePath = "/contents", basePackage = "static")
```

Comment:
Nice! Didn't know about this one.

Reply:
The old method was deprecated in the latest version of ktor so I switched to the new way to do it.